app/meldingen/admin/containers/IncidentDetailPage/reducer.js

```diff
@@ -0,0 +1,35 @@
+/*
+ *
+ * IncidentDetailPage reducer
+ *
+ */
+
+import { fromJS } from 'immutable';
```

Comment:
Ik ben zelf nog niet helemaal over uit wat ik van immutable moet vinden. Het is in principe nice omdat je weet dat iets "immutable" is maar die syntax die het met zich mee brengt. Ik zie ook in selectors dat je nu een `.get()` doet bijv.